war/src/main/java/io/lumeer/engine/controller/CollectionMetadataFacade.java

```diff
@@ -184,26 +182,71 @@ public String createInternalName(String originalCollectionName) throws UserColle
       return name;
    }
 
+   public List<io.lumeer.engine.api.dto.Collection> getCollectionsMetadata(String collectionNamePart) {
```

Comment:
Because there is import of `LumeerConst.Collection` and it causes conflict, but I will refactor this import to be more cleaner what is going on...